.github/workflows/main.yml

```diff
@@ -140,6 +140,36 @@ jobs:
       - name: Test build on ${{ env.TOOLCHAIN }} toolchain
         run: ${{ matrix.task }}
 
+  checks-with-ci-repo:
+    runs-on: ubuntu-18.04
```

Comment:
use
```
strategy:
  matrix:
```

approach to set up a two tasks in this job: one for check-fedora-version and one for check-fedora-version-sys.

Reply:
Resolved.